skins/oasis/css/core/responsive-core.scss

```diff
@@ -13,19 +13,23 @@
 		min-width: $width-outside-min;
 	}
 
-	.skin-oasis > .global-notification,
+	.skin-oasis > .banner-notifications-wrapper,
 	.WikiaHeader .page-width-container,
 	.WikiaPage {
 		max-width: $width-outside-max;
 	}
 
-	.global-notification,
+	.banner-notifications-wrapper,
 	.WikiaHeader .page-width-container {
 		@include box-sizing(border-box);
 		width: $width-outside;
 	}
 
-	.global-notification {
+	.banner-notification {
+		@include box-sizing(border-box);
```

Comment:
We don't need a mixin for this anymore. `box-sizing: border-box` works on all supported browsers now. :)
http://caniuse.com/#search=box-sizing

Someone should really clean that up...